src/NUnitFramework/framework/Schemas/TestResult.xsd

```diff
@@ -215,6 +215,7 @@
       <xs:attribute name="inconclusive" type="NonnegativeInt32" use="required" />
       <xs:attribute name="skipped" type="NonnegativeInt32" use="required" />
       <xs:attribute name="asserts" type="NonnegativeInt32" use="required" />
+      <xs:attribute name="warnings" type="NonnegativeInt32" use="required" />
```

Comment:
I know that ordering of attributes does not matter, but can we move the "warnings" line one line up, i.e. above 
```xml
<xs:attribute name="asserts" type="NonnegativeInt32" use="required" />
```
so that the xsd and the code in `NUnit3XmlOutputWriter.cs` have the same order of attributes

Reply:
Done